packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-figure-aria.tsx

```diff
@@ -13,15 +13,23 @@ const {InfoTip} = components;
 
 type Props = {
     ariaLabel: string | undefined;
-    prePopulatedAriaLabel: string;
+    prePopulatedAriaLabel?: string;
+    getPrepopulatedAriaLabel?: () => Promise<string>;
     onChangeProps: (props: {ariaLabel?: string | undefined}) => void;
```

Comment:
Can this be moved up since `prePopulatedAriaLabel` and `getPrepopulatedAriaLabel` are optional?

Reply:
I believe we have a general style of putting function type props at the end of the props list in all the Khan repos.